### PR TITLE
65 Removing Redux Usage for basic UI components

### PR DESCRIPTION
**Pull request by ChiliPaneer**
Using props to avoid unnecessarily using redux to open the student sidebar and quick view. Currently in progress - dealing with weird bug when selecting first student.

### CODE REVIEW

**Comment by ChiliPaneer** [comment]
I can't really test this because there are no students showing up locally for me.

File: pharmd-app/src/screens/student/StudentDrawer.js [hunk 1]
```diff
@@ -83,97 +84,113 @@ transition: ${props =>
...
+
+
+// Avoid route errors
+    const quickview = () => {
+      return isOpenMatch ? (
+        <>
+          <StudentQuickView id={selected}  {...props} />
+          <ButtonSpan>
+            <RouterLink to={`/students/${props.id}/details`}>
+              <DeatilsButton>More Student Info</DeatilsButton>
```
Comment (sophiacrennan) [inline pharmd-app/src/screens/student/StudentDrawer.js]: There's a typo here. It should be `DetailsButton`. Otherwise, everything still works!

**Review by sophiacrennan — COMMENTED** [review]
Looks good! Just make sure to fix the typo in StudentDrawer and remove all the commented out code. If you need help seeding the data so data shows up for you, just let me know.

**Review by sophiacrennan — APPROVED** [review]
Looks good!

**Pull request merged**